### PR TITLE
Export metrics to InfluxDB at a configurable interval

### PR DESCRIPTION
**Pull request by isaachili**
## Changes

`InfluxDBExporterExtensions.AddInfluxDBMetricsExporter(MeterProviderBuilder, Action<InfluxDBMetricsExporterOptions>)` uses a `PeriodicExportingMetricReader` instance to pass the metrics to the InfluxDB exporter at an interval.  However, the extension method provides no means to make this interval user-configurable.

This PR introduces a new property, `MetricExportIntervalMilliseconds`, to `InfluxDBMetricsExporterOptions` to configure the said interval.  The property value is then passed as an argument to the `exportIntervalMilliseconds` parameter in the `PeriodicExportingMetricReader` constructor.

The default value for `InfluxDBMetricsExporterOptions.MetricExportIntervalMilliseconds` is set to 60000, matching the default that's specified with `PeriodicExportingMetricReader.DefaultExportIntervalMilliseconds`.

### CODE REVIEW

**Comment by Kielek** [comment]
@Havret  fyi

**Review by cijothomas — APPROVED** [review]
change looks good! Please add a changelog entry.

**Comment by Kielek** [comment]
Please add new property to PublicAPi file. The easiest way - by fix command for each target framework in Visual Studio,

**Comment by Havret** [comment]
Besides missing bits in PublicAPi file and changelog, LGTM.

**Review by Havret — APPROVED** [review]

File: src/OpenTelemetry.Exporter.InfluxDB/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt
Comment (utpilla) [inline src/OpenTelemetry.Exporter.InfluxDB/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt:1]: @isaachili Add these entries for net6.0 public API files as well.

**Comment by isaachili** [comment]
Do you have a rough idea on when this will be released? Thanks

**Comment by Kielek** [comment]
@isaachili, please follow "Pre-steps" from https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/build/RELEASING.md#pre-steps and create PR. 

Version probably `1.0.0-alpha.3`. Previous release PR: #1246

**Comment by isaachili** [comment]
Thanks! PR opened: #1398

**Pull request merged**